src/lib.rs

```diff
@@ -132,8 +169,10 @@ impl log::Log for Logger {
     fn log(&self, record: &log::Record) {
         if self.enabled(record.metadata()) {
             match self.level {
-                log::Level::Trace => self.log_with_trace(record),
-                _ => self.log_with_level(record),
+                log::Level::Trace => self
+                    .log_with_trace(record)
+                    .expect("Failed to log with trace"),
+                _ => self.log_with_level(record).expect("Failed to log"),
```

Comment:
The `termcolor` crate returns a result for all their methods because writing using `std::io::Write` returns `std::io::Result<()>`. In this case data is only written to in-memory buffers so this should never fail in practice.